contracts/modules/capital/SwapOperator.sol

```diff
@@ -295,6 +314,129 @@ contract SwapOperator is ReentrancyGuard {
     emit Swapped(ETH, stETH, amountIn, amountOut);
   }
 
+  function swapETHForEnzymeVaultShare(uint amountIn, uint amountOutMin) external onlySwapController {
+    IPool pool = _pool();
+    IEnzymeV4Comptroller comptrollerProxy = IEnzymeV4Comptroller(IEnzymeV4Vault(enzymeV4VaultProxyAddress).getAccessor());
+    IERC20Detailed toToken = IERC20Detailed(enzymeV4VaultProxyAddress);
+    IWETH weth = IWETH(router.WETH());
```

Comment:
`router` will be removed in V2. Let's add WETH as an immutable. This will also avoid an extra external call.